test/controllers/employer.test.ts

```diff
@@ -110,10 +109,35 @@ describe('user controller', () => {
       .post('/employer/jobs/google')
       .send(jobPost)
 
-    console.log(response.body)
     expect(response.status).toBe(200)
     expect(response.body.message).toBe('Posted')
   })
 
-  
+  it('should update job post', async () => {
+    const jobPost = {
+      title: 'Fullstack React- & Node.js Developer',
+      jobDescription:
+        'We create and operate the online shops of Klamotten. Your job is to participate in the further development of our existing shop system platform',
+      seniority: 'Junior',
+    }
+
+    await createEmployer()
+
+    const response = await request(app)
+      .post('/employer/jobs/google')
+      .send(jobPost)
+
+    const jobPostId = response.body.savedJobPost.id
+
+    const update = {
+      title: 'Updated job title',
+      jobDescription: 'Updated Job Description',
+      seniority: 'Junior',
+    }
```

Comment:
Can you improve this by update only what is included in the update form

Reply:
Improvement made on the update form. As discussed, await Skill table creation, thereby omitting it for now.